datasets/batch_view/main_summary/reference.md

```diff
@@ -452,6 +452,21 @@ Most of the fields are simple scalar values, with a few notable exceptions:
 * Dynamically-included histogram fields are present as key->value maps,
   or key->(key->value) nested maps for keyed histograms.
 
+## Time formats
+
+Columns in `main_summary` may use one of a handful of time formats with different precisions:
+
+Column Name              | Origin                                   | Description                                  | Example                         | Spark                                                                    | Presto
+-------------------------|------------------------------------------|----------------------------------------------|---------------------------------|--------------------------------------------------------------------------|---------------------------------------------------------------------------
+`timestamp`              | stamped at ingestion                     | nanoseconds since epoch                      | `1504689165972861952`           | `from_unixtime(timestamp/1e9)`                                           | `from_unixtime(timestamp/1e9)`
+`submission_date`        | derived from timestamp                   | `YYYYMMDD` date string of timestamp          | `20170906`                      | `from_unixtime(unix_timestamp(submission_date, 'yyyyMMdd'))`             | `date_parse(submission_date, '%Y%m%d')`
```

Comment:
drive-by: please change this to `submission_date_s3` - it is the same format, but is preferred since it is a partitioning column and will result in faster querying.

Reply:
:+1:; thanks!